src/wallet/test/init_tests.cpp

```diff
@@ -73,8 +73,6 @@ BOOST_AUTO_TEST_CASE(walletinit_verify_walletdir_no_trailing)
     BOOST_CHECK_EQUAL(walletdir, expected_path);
 }
 
-#ifndef WIN32
-// Windows does not consider "datadir/wallets//" to be a valid directory path.
 BOOST_AUTO_TEST_CASE(walletinit_verify_walletdir_no_trailing2)
 {
     SetWalletDir(m_walletdir_path_cases["trailing2"]);
```

Comment:
Is this going to cause any read or write outside the m_root_path?

Reply:
> Is this going to cause any read or write outside the m_root_path?

I don't think so, but I'm also not sure where the question is coming from. Wouldn't you expect invalid paths to trigger errors, not to read or write to completely different locations?